parm/aero/jcb-base.yaml.j2

```diff
@@ -137,3 +137,6 @@ bias_files:
   ssmis_f18: rad_varbc_params.tar
```

Comment:
I suspect all these lines that end with `rad_varbc_params.tar` can be removed from the `parm/aero/jcb-base.yaml.j2` file since they will be used (and were copied from) the `parm/atm` version

Reply:
Right, I will update and test it.